packages/e2e/local-network/scripts/nodes-configuration.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+AMOUNT_PER_NODE='10000000000000'
+NUM_SPO_NODES=11
```

Comment:
Increasing # of nodes might require to also increase # of transactions in delegation rewards test, as it lowers the probability of generating rewards

Reply:
Let's keep it as-is, we'll know what to do if we see it failing with 0 rewards